styles.css

```diff
@@ -0,0 +1,88 @@
+.container {
+  background: #1c1a19;
+  width: 375px;
+  height: 662px;
+  left: 500px;
+  top: 38px;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+}
+
```

Comment:
- [ ] Please add this graphic decoration on the bottom-right corner as it is in the design given. This improves the aesthetics of your UI 😉 
Figma
![image](https://user-images.githubusercontent.com/90222110/191987193-bf38b428-33f2-46a3-b828-e1bebc63529a.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/191987331-e535e6f7-e2ae-458e-906a-a41b8dd46dcd.png)